authors/apps/profiles/models.py

```diff
@@ -15,11 +15,11 @@ class Profile(models.Model):
                                 'description about yourself.')
     name = models.CharField(max_length=50, help_text='Enter your first '
                             'and last names.')
-    number_of_followers = models.IntegerField(default=0)
-    number_of_followings = models.IntegerField(default=0)
-    total_articles = models.IntegerField(default=0)
+    number_of_followers = models.IntegerField(default=150)
+    number_of_followings = models.IntegerField(default=100)
```

Comment:
I believe the number of followers and following should default to zero as initially, the user has none.

Reply:
They are just placeholders since the fields have been put to read only to avoid any editing in the future feature of the implementation, they will be set accordingly

Reply:
I do see your concern however, let make the adjustments.

Reply:
@Raywire Kindly confirm that the changes have been made.